aws-auth-cognito/src/test/java/com/amplifyframework/auth/cognito/helpers/SessionHelperTests.kt

```diff
@@ -34,6 +36,9 @@ class SessionHelperTests {
         expiration = 0
     )
 
+    // Encoded Date Time: April 30, 2024 02:00AM
+    private val currentInstant = Instant.ofEpochSecond(1714467600)
```

Comment:
No longer needed.

Reply:
Can't believe ktlint didn't catch that 🤦